(node/lsstcam-pcs01.cp.lsst.org) provision host EL9 & add tests

badenerb opened the pull request.

Commit:
(node/lsstcam-pcs01.cp.lsst.org) add network config & add test

dtapiacl reviewed (changes requested):
Please add the node to IPAM and binded to a device.

jhoblitt reviewed (changes requested):
Please squash the commits together, add a device to ipam, and associate IP addresses with the device.

jhoblitt commented on spec/hosts/nodes/lsstcam-pcs01.cp.lsst.org_spec.rb:
```diff
@@ -0,0 +1,57 @@
+
+describe 'lsstcam-pcs01.cp.lsst.org', :sitepp do
+  on_supported_os.each do |os, os_facts|
+    next if os =~ %r{centos-7-x86_64}
```
```suggestion
    next unless os =~ %r{almalinux-9-x86_64}
```

glennmorris committed:
(node/lsstcam-pcs01.cp) add ccs services chiller, refrig

(node/lsstcam-vw05.ls.lsst.org) provision host as EL9 & add test

(node/lsstcam-pcs01.cp.lsst.org) provision host as EL9 & add test

jhoblitt commented on spec/hosts/nodes/lsstcam-pcs01.cp.lsst.org_spec.rb:
```diff
@@ -0,0 +1,57 @@
+        it { expect(nm_keyfile['ipv4']['address1']).to eq('139.229.175.94/26,139.229.175.126') }
+        it { expect(nm_keyfile['ipv4']['dns']).to eq('139.229.160.53;139.229.160.54;139.229.160.55;') }
+        it { expect(nm_keyfile['ipv4']['dns-search']).to eq('cp.lsst.org;') }
+        it { expect(nm_keyfile['ipv4']['method']).to eq('manual') }
```
```suggestion
        it_behaves_like 'nm manual interface'
```

Using the shared example makes refactoring slightly easy in the future and it currently test slightly more than this line but the man reason to use the shared example is consistency with the other spec files.

jhoblitt commented:
I believe there is an extra unintended commit on this branch that should be removed: ea34804b7a591ffa71540142eed06ede312c23b5

dtapiacl reviewed (approved):
LGTM

jhoblitt commented on hieradata/node/lsstcam-pcs01.cp.lsst.org.yaml:
```diff
@@ -0,0 +1,37 @@
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "192.168.1.1/24"
```
Does this conflict with lsstcam-ccobwb or is this a different network segment?

https://ipam.cp.lsst.org/index.php?page=subnets&section=1&subnetId=144&sPage=address-details&ipaddrid=5406

badenerb replied:
@jhoblitt It shouldn't. From what I understand this is a private ip that will only talk to the other chiller units. It won't be connected to our network at all.

jhoblitt reviewed (changes requested):
This generally looks good but the commits have the title copied unnecessarily into the commit message body  ![image](https://github.com/lsst-it/lsst-control/assets/948901/0a4b91e8-a725-4bf4-9e7d-5e630854d8b6).

jhoblitt reviewed (approved):
LGTM.
